cmds/core/time/time_test.go

```diff
@@ -2,44 +2,43 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build !tinygo
+// +build !tinygo
+
 package main
 
 import (
 	"bytes"
+	"errors"
+	"os/exec"
 	"regexp"
 	"testing"
 )
 
 func TestTimeRewrite(t *testing.T) {
```

Comment:
Looks like I forgot to remove `Rewrite` in test function name after my latest change.

Reply:
done